tests/try_scraper/test_invalid_revision.py

```diff
@@ -5,24 +5,18 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from mock import patch
-
+import pytest
 from mozdownload import TryScraper
 import mozdownload.errors as errors
-import mozhttpd_base_test as mhttpd
-
 
-class TestTryScraperInvalidParameters(mhttpd.MozHttpdBaseTest):
-    """test mozdownload TryScraper class with invalid parameters"""
 
-    @patch('mozdownload.treeherder.Treeherder.query_builds_by_revision')
-    def test_scraper(self, query_builds_by_revision):
-        """Testing download scenarios with invalid parameters for TryScraper"""
-        query_builds_by_revision.return_value = []
+@patch('mozdownload.treeherder.Treeherder.query_builds_by_revision')
+def test_scraper(query_builds_by_revision, httpd, tmpdir):
```

Comment:
Given that we loose the class name, this test needs a clear name now. Also lets add the patch argument as last one so that fixtures always come first.